lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -1542,75 +1542,18 @@
   ],
   "Accessibility": {
     "violations": [
-      {
```

Comment:
I'm thinking this went away because the smoke test page has since been "fixed" of a color contrast issue. Or maybe the axe violation check was modified since last time this ran.

Do we want to resolve this?

Reply:
Given it was a consequence of us rebaselining and not of this version bump, I'm not concerned. IMO, the goal here of sample artifacts is to test at least *some* of the axe-audits and it's not particularly important which ones get exercised.

Reply:
it is still failing on stable but not in Canary. Looks like the root cause is the text color styling via `/deep/` but that not being supported anymore:

https://github.com/GoogleChrome/lighthouse/blob/c6e14d7c8629c17284c1f773dd6bc8bc61221a47/lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html#L109-L111

Reply:
Nice find! Do you agree it's not a blocker here @brendankenny ?

Reply:
Yes. We should probably update at some point just because that `/deep/` does nothing (I think we also previously removed the deprecation notice it used to generate because Chrome doesn't even bother anymore), but agreed that's more a smoke test thing, not the purpose of the sample artifacts/sample_v2